docs/persistent_compilation_cache.md

```diff
@@ -67,8 +67,135 @@ Cloud Storage (GCS) bucket. We recommend the following configuration:
 Assuming that `gs://jax-cache` is the GCS bucket, set `cache-location` as
 follows:
 
-```
+```python
 import jax
 
 jax.config.update("jax_compilation_cache_dir", "gs://jax-cache")
```

Comment:
Should mention that `gs://` does not work automatically. It requires `etils[epath]` to be installed.

Should mention that `etils[epath]` supports many cloud providers, while GCS is one common choice.